### PR TITLE
(DOCSP-23676): Source constant, admonition, term cleanup

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-23676

### Staged Changes (Requires MongoDB Corp SSO)

- [Realm SDK Docs](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-23676/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by mongodben — COMMENTED** [review]
back to you!

File: source/sdk/node/advanced/multi-user-applications.txt [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following states describe an on-device user at any given time:
 - **Logged Out:** any user that authenticated on the device but
   has since logged out or had their session revoked.
 
-The following diagram shows how users within a {+service-short+} client app
+The following diagram shows how users within a Realm client app
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/sdk/node/advanced/multi-user-applications.txt:57]:
```suggestion
The following diagram shows how users within an App Services client app
```

**Review by mongodben — COMMENTED** [review]
i still have some reservations about full destruction of the glossary. i think there's valuable info in there. but i think keeping it 'siloed' in the glossary doesnt make sense. 

here's what i propose instead:
- move the glossary to the root folder (not in source directory) so we have a record of it in the repo still
- create a new ticket for me to add the glossary information throughout the docs in relevant locations. i'll figure out what that means and how to do as part of that unit of work.

**Comment by nathan-contino-mongo** [comment]
@mongodben https://jira.mongodb.org/browse/DOCSP-23835
Created!

**Review by mongodben — APPROVED** [review]
lgtm!

**Pull request merged**